README.md

```
@@ -26,34 +26,50 @@ normative design and roadmap live as issues in the private tracker.
+    opus:
+      bitrate: 64000
+  - name: ultrasonic-mic   # add as many devices as the hardware supports
+    device: "hw:2,0"
+    path: /bat
+    mode: pcm
+    rate: 256000
+    channels: 1
+    format: s16
 ```
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)